ted/src/main/java/frc/robot/common/CanUtils.java

```diff
@@ -0,0 +1,104 @@
+package frc.robot.common;
+
+import com.ctre.phoenix.motorcontrol.StatusFrameEnhanced;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.playingwithfusion.TimeOfFlight;
+import com.playingwithfusion.TimeOfFlight.RangingMode;
+import frc.robot.control.HardwareConstants;
+
+/**
+ * A class intending to help keep CAN usage tidy
+ */
+public class CanUtils {
+
+    /**
+     * A method to update the TalonFX motor frequency to team-wide approved setting
+     * @param motor - the TalonFX motor (usually not a follower motor)
+     */
+    public static void UpdateCtreFrequency(TalonFX motor) {
```

Comment:
I think having a function that overrides previous settings will cause a lot of confusion. There could be instance where a setting is made in one place, and then overridden using this function in a different place. If the motor is not behaving as expected, it would be very hard to track down why. 
I would prefer this to be the single place where we set update frequencies. It should take in a desired frequency and warn if you set a frequency that's higher than the recommended rate. 
Then, we could enforce through coder review that people always go through this helper to set update rates.

Reply:
This is not used in the PR, I had included it just in the case we needed more than just the changes to swerve stuff (motors and encoders) was not sufficient and we needed more.  The file is now removed.  If we end up needing it back we will bring it from the change history.